src/SprykerEco/Yves/Easycredit/Controller/AbstractController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Easycredit\Controller;
+
+use Spryker\Yves\Kernel\Controller\AbstractController as SprykerAbstractController;
+
+abstract class AbstractController extends SprykerAbstractController
+{
+    /**
+     * @var array
+     */
+    protected $responseArray;
+
+    /**
+     * @return void
+     */
+    public function initialize()
+    {
+        parent::initialize();
+
+        $this->responseArray = $this->getApplication()['request']->query->all();
```

Comment:
I would perform convert during this loading, so our code will receive proper transfer object